Resources/views/pjax_admin.html.twig

```diff
@@ -25,3 +25,19 @@
     </header>
     {% block content %}{% endblock %}
 </div>
+
+{% block notification %}
+{# Possible states are "error", "started", and "done" #}
+<div data-name="notification" data-state="{{ notification_state|default("done") }}">
+    {% set notifications = app.session.flashBag.get("notification") %}
+    {% if notifications|length == 1 %}
+        {{ notifications[0] }}
```

Comment:
actually this comment makes me think we need to think a bit more of the case where we have several messages vs. the case where there's only one.